examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/OfferingFragment.kt

```diff
@@ -270,21 +265,15 @@ class OfferingFragment : Fragment(), PackageCardAdapter.PackageCardAdapterListen
             .show()
     }
 
-    private fun showProrationModePicker(callback: (Int?) -> Unit) {
-        val prorationModeOptions = mapOf(
-            0 to "None",
-            ProrationMode.IMMEDIATE_WITH_TIME_PRORATION to ProrationMode::IMMEDIATE_WITH_TIME_PRORATION.name,
-            ProrationMode.IMMEDIATE_AND_CHARGE_PRORATED_PRICE to
-                ProrationMode::IMMEDIATE_AND_CHARGE_PRORATED_PRICE.name,
-            ProrationMode.IMMEDIATE_WITHOUT_PRORATION to ProrationMode::IMMEDIATE_WITHOUT_PRORATION.name,
-            ProrationMode.DEFERRED to ProrationMode::DEFERRED.name,
-            ProrationMode.IMMEDIATE_AND_CHARGE_FULL_PRICE to ProrationMode::IMMEDIATE_AND_CHARGE_FULL_PRICE.name
-        )
-        @ProrationMode var selectedProrationMode = 0
+    private fun showProrationModePicker(callback: (GoogleProrationMode?) -> Unit) {
+        val prorationModeOptions = GoogleProrationMode.values()
+        var selectedProrationMode = GoogleProrationMode.IMMEDIATE_WITHOUT_PRORATION
```

Comment:
I think we should avoid specifying the default here as well. Ideally we only set the default in a single place. Since this method supports returning a null proration mode, I think we can just default this to null. Thoughts?

Reply:
yeah sure! that will help us test our defaulting logic too. good point.